packages/data-point/lib/reducer-types/reducer-list/factory.test.js

```diff
@@ -33,7 +33,7 @@ describe('factory#create', () => {
     expect(reducer.asCollection).toBe(true)
   })
 
-  test('factory#create context with reducers', () => {
+  test('factory#create context with reducers | using piped reducer', () => {
```

Comment:
Can we remove the `|` but leave the stuff after?

`It should create a ReducerList with reducers using a piped reducer`?